.github/workflows/main.yaml

```
@@ -7,56 +7,12 @@ on:
     branches: master
 
```

With upstream dev test runs, it might be useful to add nightly test runs

```suggestion
 schedule:
    - cron: "0 0 * * *"
```

Thoughts?

---

Let's give it a try!